### PR TITLE
[IMP] *_*: update  tooltips in various modules

### PR DESCRIPTION
**Pull request by visp-odoo**
In this commit we rewrite some tooltips   because some of them are not
correct in English and others are not valid anymore.Add some tooltips that
could be missing and remove some tooltips that are not valid anymore

### CODE REVIEW

File: addons/project/models/project.py [hunk 1]
```diff
@@ -74,32 +74,36 @@ def _get_default_project_ids(self):
...
+        help="Name of your stage e.g. To do")
     description = fields.Text(translate=True)
-    sequence = fields.Integer(default=1)
+    sequence = fields.Integer(default=1,
+        help="Determines the order in which stages are displayed.")
     project_ids = fields.Many2many('project.project', 'project_task_type_rel', 'type_id', 'project_id', string='Projects',
-        default=_get_default_project_ids)
+        default=_get_default_project_ids,
+        help="Projects in which this stage is present. If you follow a similar workflow in several projects \n"
+            "You can share this stage among them and get consolidated information this way.")
```
Comment (xavierbol) [inline addons/project/models/project.py:85]:
```suggestion
        help="Projects in which this stage is present. If you follow a similar workflow in several projects,\n"
            " you can share this stage among them and get consolidated information this way.")
```
I don't know if the `\n` is really needed.

File: addons/sale_project/models/project.py [hunk 2]
```diff
@@ -12,13 +12,12 @@
...
 
-    allow_billable = fields.Boolean("Billable", help="Invoice your time and material from tasks.")
+    allow_billable = fields.Boolean("Billable")
     sale_line_id = fields.Many2one(
         'sale.order.line', 'Sales Order Item', copy=False,
         compute="_compute_sale_line_id", store=True, readonly=False, index='btree_not_null',
         domain="[('is_service', '=', True), ('is_expense', '=', False), ('state', 'in', ['sale', 'done']), ('order_partner_id', '=?', partner_id), '|', ('company_id', '=', False), ('company_id', '=', company_id)]",
-        help="Sales order item to which the project is linked. Link the timesheet entry to the sales order item defined on the project. "
-        "Only applies on tasks without sale order item defined, and if the employee is not in the 'Employee/Sales Order Item Mapping' of the project.")
+        help="Sales order item that will be selected by default on the tasks and timesheets of this project, except if the employee set on the timesheets is explicitely linked to another sales order item on the project. It can be modified on each task and timesheet entry individually if necessary.")
```
Comment (xavierbol) [inline addons/sale_project/models/project.py:20]: Sometimes you split the line in more when the sentence is too long and sometimes you put some `\n`. 🤔 
Why does the sentence here has no `\n`?

File: addons/sale_project/models/project.py [hunk 3]
```diff
@@ -471,7 +470,9 @@ class ProjectTask(models.Model):
         copy=True, tracking=True, index='btree_not_null', recursive=True,
         compute='_compute_sale_line', store=True, readonly=False,
         domain="[('company_id', '=', company_id), ('is_service', '=', True), ('order_partner_id', '=?', partner_id), ('is_expense', '=', False), ('state', 'in', ['sale', 'done'])]",
-        help="Sales Order Item to which the time spent on this task will be added, in order to be invoiced to your customer.")
+        help="Sales Order Item to which the time spent on this task will be added in order to be invoiced to your customer."
+             "By default the sales order item set on the project will be selected. In the absence of one, the last prepaid sales order item that has time remaining will be used."
+             "Remove the sales order item in order to make this task non billable. You can also change or remove the sales order item of each timesheet entry individually.")
```
Comment (xavierbol) [inline addons/sale_project/models/project.py:475]: No need `\n` here? 🤔

**Review by xavierbol — COMMENTED** [review]
Thanks for your work, I don't understand why sometimes you added some `\n` and sometimes not. 🙂

File: addons/sale_project/models/project.py (hunk 3 above)
Comment (mba-odoo) [reply addons/sale_project/models/project.py]: @xavierbol , Yes, needed over here, Done all the changes. Thanks

**Review by IT-Ideas — APPROVED** [review]
@robodoo r+
Thanks 🙂

**Comment by IT-Ideas** [comment]
@robodoo r+

**Pull request closed without merging**